packages/braid-design-system/CHANGELOG.md

```diff
@@ -1,5 +1,432 @@
 # braid-design-system
 
+## 33.0.0
+
```

Comment:
```suggestion
## 33.0.0

This release is a huge milestone for Braid with upgrades occurring across a number of key areas:

### Migrate layout components to [CSS Gap]
With SEEKs Browser Support Policy now enabling adoption of [CSS gap], Braids layout components are now able to lean into the platform directly for its declarative, parent-driven approach to white space management.

The result is on average our experiences receive around a 20% reduction in DOM elements, more performant rendering, better composability without the introduction of intermediary elements, a better debug experience in dev tools.

The key tradeoff is the removal of `dividers` functionality which is explained further [here].

### Other key changes
- Remove support for React v17.x
- Removal of deprecated features
- Reduce `gutter` size in `seekJobs` theme

[CSS Gap]: https://developer.mozilla.org/en-US/docs/Web/CSS/gap
[here]: https://github.com/seek-oss/braid-design-system/blob/8177e4c6b502536e8f37811f5eef735ff265f1c6/docs/Removing%20dividers%20support%20from%20layout%20components.md

See full changelog below 👇 

```

Reply:
- `Braids layout components` -> `Braid's layout components`
- `, a better debug experience` -> `and a better debug experience`

Reply:
Not 100% on this phrasing `The result is on average our experiences recieve around a 20% reduction...`. Maybe `The result on our experiences is, on average, around a 20% reduction..` or something to that effect.